sandbox/technical_user_guide.md

```diff
@@ -332,479 +351,249 @@ The dollar sign (‘$’) before the word “export” in the URL indicates that
 * `Accept: application/fhir+json`
 * `Prefer: respond-async`
 
-**cURL command**
+**`cURL` command**
 ```
-curl -X GET "https://sandbox.bcda.cms.gov/api/v1/Patient/$export" -H "accept: application/fhir+json" -H "Prefer: respond-async" -H "Authorization: Bearer {token}"
+curl -X GET "https://api.bcda.cms.gov/api/v1/Patient/$export"
+-H "accept: application/fhir+json"
+-H "Prefer: respond-async"
+-H "Authorization: Bearer {token}"`
 ```
 
 **Response**
 
-If the request was successful, a `202 Accepted` response code will be returned and the response will include a *Content-Location* header. The value of this header indicates the location to check for job status and outcome. In the example header below, the number 42 in the URL represents the ID of the export job.
+If the request was successful, a `202 Accepted` response code will be returned and the response will include a _Content-Location_ header. The value of this header indicates the location to check for job status and outcome. In the example header below, the number `42` in the URL represents the ID of the export job.
 
 **Headers**
 
-* `Content-Location: https://sandbox.bcda.cms.gov/api/v1/jobs/42`
+* `Content-Location: https://api.bcda.cms.gov/api/v1/jobs/42`
```

Comment:
Should this be sandbox?

Reply:
I see lots of instances where `api` is referenced instead of `sandbox`.  Can you change all of them?